lerobot/common/policies/factory.py

```diff
@@ -95,23 +98,23 @@ def make_policy(
         raise ValueError(
             "Exactly one of `pretrained_policy_name_or_path` and `dataset_stats` must be provided."
         )
-
+    precision = torch.float16 if "fp16" in hydra_cfg.precision else torch.float32
```

Comment:
Here if you are loading older checkpoints, it throws an error. 

`omegaconf.errors.ConfigAttributeError: Key 'precision' is not in struct
    full_key: precision
    object_type=dict`
    
    So, I force  it to be `precision=None` if I the older checkpoints didn't have this arg.
    Again, backward compatibility issue. Do you think abstracting `precision` from training config and leave it only inside vla related config will solve the issue?

Reply:
Yes having this for vla is a good option